README.md

```diff
@@ -156,25 +173,50 @@ See the top-level `package.json` "scripts" section. Some common targets are:
 
 The Taquito integration tests are located in the `integration-tests/` directory.
 
-To run the integration tests, `cd` into `integration-tests/` and run `npm run test` (ensure you have completed the build steps as described earlier in this README file.). The integration test suite runs all tests against the current tezos protocol testnet, and typically also against the previous and next protocol testnets. See the `scripts` property in the `integration-tests/package.json` file for specific test targets.
+To prepare to run the integration tests, perform the following steps:
```

Comment:
These instructions are for running the integration tests with flextesa, but it is also possible to run them with a public node. There are instructions in the integration-tests/README.md file as well to run the integration tests. Should everything be consolidated in one place?

Reply:
Woops, I "forgot" about the README in `integration-tests`.
Yes, this should be consolidated there. We should provide instructions to run against both environments.
I will add a placeholder in the top-level README to go look at `integration-tests/README` to learn how to run them. I will move the instructions @danielelisi kindly helped me with to that location at the same time.